avocado/core/data_dir.py

```diff
@@ -101,18 +101,19 @@ def get_test_dir():
     The heuristics used to determine the test dir are:
     1) If an explicit test dir is set in the configuration system, it
     is used.
-    2) If user is running Avocado out of the source tree, the example
-    test dir is used
-    3) System wide test dir is used
-    4) User default test dir (~/avocado/tests) is used
+    2) If user is running Avocado in the source tree, the example test dir is
```

Comment:
I think the original intention here was something like "running a business out of a van" (a food truck), in the sense that it operated from that location.  Maybe the best wording is "If a user is running Avocado from its source code tree...".  Just a suggestion though.

Reply:
Sure, I'll update it.